sdk/python/tests/dsl/utils_tests.py

```diff
@@ -0,0 +1,66 @@
+from kfp.dsl import ContainerOp
```

Comment:
Can you also add this to `main.py`? Thanks!

Reply:
Will do, didn't actually understand how the tests where run  :)

Reply:
I have added to main.py but there is alos a .sh file which runs unittest, but then the test need to be named test_something.py as I understand it?

Reply:
That's correct. The one with the path `sdk/python/tests/dsl/test_utils.py` should suffice. Then `python3 -m unittest discover` can automatically load it.